classes/controller/FrontController.php

```diff
@@ -1549,7 +1549,10 @@ public function getTemplateVarConfiguration()
 
     protected function getDisplayTaxesLabel()
     {
-        return (Module::isEnabled('ps_legalcompliance') && (bool) Configuration::get('AEUC_LABEL_TAX_INC_EXC')) || $this->context->country->display_tax_label;
+        return
+            (Module::isEnabled('ps_legalcompliance') && (bool) Configuration::get('AEUC_LABEL_TAX_INC_EXC'))
+            || (Configuration::get('PS_TAX') && $this->context->country->display_tax_label)
```

Comment:
Also we might need to cast it into bool 🤔I see this has been done for `AEUC_LABEL_TAX_INC_EXC'`